.vscode/c_cpp_properties.json

```diff
@@ -90,4 +90,4 @@
         }
     ],
     "version": 4
-}
```

Comment:
Could you revert this change?

Reply:
Should be resolved now. The github editor seemed to have been auto adding the newline, so sme of my commits have no changes because of it.